input/resources/ValueSet-ehealth-program.json

```diff
@@ -0,0 +1,31 @@
+{
+  "resourceType": "ValueSet",
+  "id": "ehealth-program",
+  "url": "http://ehealth.sundhed.dk/vs/ehealth-program",
+  "version": "2024-11-28T09:00:00",
+  "name": "Program",
+  "title": "Program",
+  "status": "active",
+  "experimental": true,
+  "date": "2024-11-28T09:00:00+00:00",
+  "publisher": "ehealth.sundhed.dk",
+  "contact": [
+    {
+      "name": "ehealth.sundhed.dk",
+      "telecom": [
+        {
+          "system": "url",
+          "value": "http://ehealth.sundhed.dk/terminology"
+        }
+      ]
+    }
+  ],
+  "description": "A list of program/project.",
```

Comment:
plurals

Reply:
Programs or projects using the eHealth Infrastructure and wanting to mark various resources as intended for their non-exclusive use.